packages/opencensus-exporter-stackdriver/src/types.ts

```diff
@@ -72,13 +72,13 @@ export enum MetricKind {
 }
 
 export enum ValueType {
-  VALUE_TYPE_UNSPECIFIED,
-  BOOL,
-  INT64,
-  DOUBLE,
-  STRING,
-  DISTRIBUTION,
-  MONEY
+  VALUE_TYPE_UNSPECIFIED = 'VALUE_TYPE_UNSPECIFIED',
+  BOOL = 'BOOL',
+  INT64 = 'INT64',
+  DOUBLE = 'DOUBLE',
+  STRING = 'STRING',
+  DISTRIBUTION = 'DISTRIBUTION',
+  MONEY = 'MONEY'
```

Comment:
I don't think we support "Money", "String" and "Bool". Do we still need them here?

Reply:
I don't think we need that, looks like previously copied from https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors#ValueType list.

Reply:
Removed.